src/Extension/Engine/SiteTreePublishingEngine.php

```diff
@@ -99,15 +102,16 @@ public function setToDelete($toDelete)
     }
 
     /**
-     * @param \SilverStripe\CMS\Model\SiteTree|null $original
+     * @param SiteTree|SiteTreePublishingEngine|null $original
+     * @throws ValidationException
```

Comment:
I'm not a PHPDoc expert, but this seems tautologous as it's not this method that throws it's a method that is called within it and I'd have thought that the IDEs/docs generators should be keeping track of that rather than having every single method document the exceptions that methods it calls (and then that those call and so on).